pytest.ini

```diff
@@ -0,0 +1,9 @@
+# https://docs.pytest.org/en/7.1.x/reference/reference.html#ini-options-ref
+[pytest]
+minversion = 7.4
+addopts = -ra -q
```

Comment:
May want to also add `-s -vvv` flags by default since then the output will be a bit more verbose and make it easier to see which tests actually ran.